cmd/image.go

```diff
@@ -63,7 +63,7 @@ func checkImage(container apiv1.Container, image imgFlags, result *Result) {
 	}
 
 	if contImage.name == image.name && contImage.tag != image.tag {
-		result.err = 2
```

Comment:
This should not be here!

Reply:
You mean in this PR? You are right it should have been in #26 ...

Reply:
yeah or it should be moved to different commit